tfe_hardware/qcodes_instrument_drivers/Sinko/AD5760.py

```diff
@@ -1,14 +1,26 @@
+'''
+A simple driver to control the homemade current source
+'''
+
+__author__ = "Kaushik Singirikonda"
+__email__ = 'ks105@illinois.edu'
+
 from functools import partial
 from typing import Optional, Union, Any
 import time,sys
 import clr  # install pythonnet
 
 from qcodes.instrument.parameter import DelegateParameter
 from qcodes import (Instrument, validators as vals)
-from qcodes.instrument.visa import VisaInstrument
 from qcodes.instrument.channel import InstrumentChannel
 from qcodes.utils.validators import Numbers, Bool, Enum, Ints
 
+'''
+In order to initiate the use of the current source, follow the guidelines given here :
+
+https://uillinoisedu.sharepoint.com/sites/Pfafflab/_layouts/OneNote.aspx?id=%2Fsites%2FPfafflab%2FSiteAssets%2FPfafflab%20Notebook&wd=target%28Projects%2FElementaryCircuits%2FCurrent%20Source.one%7C8C1A05E0-ED99-4899-83F1-97F9EB59041F%2F%29
+onenote:https://uillinoisedu.sharepoint.com/sites/Pfafflab/SiteAssets/Pfafflab%20Notebook/Projects/ElementaryCircuits/Current%20Source.one#section-id={8C1A05E0-ED99-4899-83F1-97F9EB59041F}&end
```

Comment:
write simplified instructions here. This code is shared with other collaborators who do not have access to our one note.

Reply:
Alright, ill do that